apps/main/src/loan/store/createScrvUsdSlice.ts

```diff
@@ -362,7 +363,7 @@ const createScrvUsdSlice = (set: SetState<State>, get: GetState<State>) => ({
 
           // invalidate user balances query
           const signerAddress = useWallet.getState().wallet?.accounts?.[0]?.address.toLowerCase()
-          queryClient.invalidateQueries({ queryKey: ['scrvUsdUserBalances', { signerAddress: signerAddress ?? '' }] })
+          invalidateScrvUsdUserBalances({ signerAddress: signerAddress ?? '' })
```

Comment:
FYI: In the other queries the `queryKey` function accepts null/undefined and the `queryFn` does not.
The validationSuite should make sure the values are valid. Note how `queryFn` has `TValidParams`:
https://github.com/curvefi/curve-frontend/blob/7783f196a44bfa0ae29a323f4b8b728bd73a171b/packages/curve-ui-kit/src/lib/types/factory.ts#L42-L44